qemu/tests/expose_host_mtu.py

```diff
@@ -56,13 +56,14 @@ def cleanup_ovs_ports(netdst, ports):
 
         if is_ovs_backend(netdst) is True:
             ports = set(get_ovs_ports(netdst).splitlines()) - \
-                    set(ports.splitlines())
+                set(ports.splitlines())
             for p in ports:
                 process.system("ovs-vsctl del-port %s %s" % (netdst, p))
 
     netdst = params.get("netdst", "switch")
     if netdst in utils_net.Bridge().list_br():
-        host_hw_interface = utils_net.Bridge().list_iface()[0]
+        host_hw_interface = \
+            utils_net.Bridge().get_structure()[netdst]['iface'][0]
```

Comment:
`get_structure()[netdst]['iface']` is similar to `def list_iface(self)`
Just a suggestion, modify `list_iface()` from avocado-vt will be better.
```python
    def list_iface(self, br=None):
        if br:
            return self.get_structure()[br]['iface']
        interface_list = []
        for br in self.list_br():
            for (value) in self.get_structure()[br]['iface']:
                interface_list.append(value)
        return interface_list
```
But this one is ok for me.

Reply:
I think it should be:
``def list_iface(self, br=None):
        interface_list = []
        if br:
            for (value) in self.get_structure()[br]['iface']:
                interface_list.append(value)
        else:
            for br in self.list_br():
                for (value) in self.get_structure()[br]['iface']:
                    interface_list.append(value)
        return interface_list
Because there might be more than one iface in the dedicated br. As long as the original list_iface returns all the iface in the br, we should keep the purpose of the code unified.
What do you think?

Reply:
Sorry, tested and your method works the same as mine but more concise. I'll create a new PR on avocado-vt, thanks.

Reply:
@xiywang Thanks your remind.

> Because there might be more than one iface in the dedicated br. 
If there are more than one interface, we need to set the mtu size for all interfaces.
This situation is almost impossible in our environment, but we can support it if you want.

Reply:
@PaulYuuu As long as you recognized this is impossible for us, I'd like to keep it this way. Thanks.

Reply:
It's OK for me.
Acked-by: Yihuang Yu <yihyu@redhat.com>